Maestro/NuGet.Config

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
```

Comment:
Guessing `<clear />` disappeared as html. I'll add it--local sources wouldn't be bad, but we might as well protect against worse problems.